python/lsst/afw/coord/refraction.py

```diff
@@ -164,9 +164,9 @@ def densityFactorDry(weather):
     waterVaporPressure = humidityToPressure(weather)
     airPressure = weather.getAirPressure()*units.pascal
     dryPressure = airPressure - waterVaporPressure
-    eqn = (dryPressure/cds.mbar)*(57.90E-8 - 9.3250E-4*units.Kelvin/temperature +
-                                  0.25844*units.Kelvin**2/temperature**2.)
-    densityFactor = (1. + eqn)*(dryPressure/cds.mbar)/(temperature/units.Kelvin)
+    eqn = float((dryPressure/cds.mbar)*(57.90E-8 - 9.3250E-4*units.Kelvin/temperature +
+                                        0.25844*units.Kelvin**2/temperature**2.))
+    densityFactor = (1. + eqn)*float(dryPressure/cds.mbar)/float(temperature/units.Kelvin)
```

Comment:
From reading the astropy docs I expected to see `value.to_value(units)`, instead of using `float(value/units)`. For example:

    dryPressure.to_value(cds.mbar)

and this seems a clear way to express what you want done.

If what you've done is a standard way to get a float in particular units, can you please point me to the appropriate docs? I realize it works, but based on reading what I found, it doesn't seem obvious to me that it should.

Reply:
Yes, that looks more clear. I'll change to `to_value(units)` throughout.